nixos/modules/services/networking/routinator.nix

```
@@ -0,0 +1,71 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.routinator;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.routinator = {
+    enable = mkEnableOption "Routinator 3000 is free, open-source RPKI Relying Party software made by NLnet Labs.";
```

This option prefixes "Whether to enable" to the given string, so I'd suggest keeping it simple.

```suggestion
    enable = mkEnableOption "Routinator 3000.";
```